hibernate-core/src/main/java/org/hibernate/cfg/DialectSpecificSettings.java

```diff
@@ -36,6 +36,15 @@ public interface DialectSpecificSettings {
 	 */
 	String ORACLE_APPLICATION_CONTINUITY = "hibernate.dialect.oracle.application_continuity";
 
+	/**
+	 * Specifies whether the dialect should use the IEEE Oracle SQL types {@code binary_float}/{@code binary_double}
+	 * over {@code float(p)}/{@code real}/{@code double precision} when generating DDL or SQL casts for float types.
+	 *
+	 * @settingDefault {@code true}
+	 * @since 7.0
+	 */
+	String ORACLE_USE_IEEE_FLOATS = "hibernate.dialect.oracle.use_ieee_floats";
```

Comment:
I would just call it `use_floating_point` or `use_floating_point_types`. Otherwise it looks like you're saying something specific about IEEE types as opposed to some other sort of floating point type.